pom.xml

```diff
@@ -1304,14 +1304,14 @@
             <plugin>
                 <groupId>io.prestosql</groupId>
                 <artifactId>presto-maven-plugin</artifactId>
-                <version>6</version>
+                <version>8</version>
```

Comment:
Per previous discussion (https://github.com/prestosql/presto/pull/1690/files#r332371929)

I am OK based on @jvanzyl explanation that this is not a requirement for provisio upgrade for server tarball.
Please extract to separate commit (maybe as a latter commit).

Reply:
Done